CIP-XXXX/README.md

```diff
@@ -0,0 +1,404 @@
+---
+CIP: ???
+Title: Cardano dApp-MutlisigWallet Web Bridge
+Status: Proposed
+Category: Wallets
+Authors: 
+  - Leo
+Implementors: NA
+Discussions:
+Created: 2023-10-12
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This document describes a webpage-based communication bridge allowing webpages (i.e. dApps) to interface with Cardano Multisig-wallets. It follows the specification format of CIP-30 as it attempts to make implementing this connector as easy as possible for applications that have already successfully implemented CIP-30. This document is a work in progress and is not yet finalized. It is expected to be updated as the ecosystem evolves. 
+
+
+## Motivation: why is this CIP necessary?
+
+In order to facilitate future dApp development, we will need a way for dApps to communicate with multisig wallets, given the unique complexities of native script based address special provisions need to be made to make the connector compatible with them.  
+
+Specifically apps building transactions need to be able to get the following information from the wallet:
+- Script descriptor 
+- Script Requirements list 
+- Collateral donator (since native script based addresses cannot provide collateral for transactions)
```

Comment:
Is this for the case when there are plutus scripts as well as native scripts present? Because native scripts don't require collateral.